static/app/components/eventsTable/eventsTable.spec.jsx

```diff
@@ -10,9 +14,53 @@ describe('EventsTable', function () {
         orgId="orgId"
         projectId="projectId"
         groupId="groupId"
+        orgFeatures={TestStubs.Organization().features}
         events={TestStubs.DetailedEvents()}
       />
     );
     expect(container).toSnapshot();
   });
+
+  it('removes replayId from the tag list when the feature is disabled', () => {
+    render(
+      <EventsTable
+        tagList={[
+          {
+            key: 'replayId',
+            name: 'Replayid',
+            totalValues: 5,
+          },
+        ]}
+        orgId="orgId"
+        projectId="projectId"
+        groupId="groupId"
+        orgFeatures={[]}
+        events={TestStubs.DetailedEvents()}
+      />
+    );
+
+    // eslint-disable-next-line jest-dom/prefer-in-document
+    expect(screen.queryByText('Replay')).toBeNull();
```

Comment:
`screen.getAllByRole('columnheader')` and `.toBeInDocument()` would be more specific assertions